addons/mass_mailing_sms/wizard/mailing_sms_test.py

```diff
@@ -12,28 +12,53 @@ class MassSMSTest(models.TransientModel):
     def _default_numbers(self):
         return self.env.user.partner_id.phone_sanitized or ""
 
-    numbers = fields.Char(string='Number(s)', required=True,
-                          default=_default_numbers, help='Comma-separated list of phone numbers')
+    numbers = fields.Text(string='Number(s)', required=True,
+                          default=_default_numbers, help='Carriage-return-separated list of phone numbers')
     mailing_id = fields.Many2one('mailing.mailing', string='Mailing', required=True, ondelete='cascade')
 
     def action_send_sms(self):
         self.ensure_one()
-        numbers = [number.strip() for number in self.numbers.split(',')]
+
+        notification_messages = []
+        numbers = [number.strip() for number in self.numbers.splitlines()]
         sanitize_res = phone_validation.phone_sanitize_numbers_w_record(numbers, self.env.user)
-        sanitized_numbers = [info['sanitized'] for info in sanitize_res.values() if info['sanitized']]
+        sanitized_numbers = [info['sanitized'] for info in sanitize_res.values() if info['sanitized'] and not info['code']]
         invalid_numbers = [number for number, info in sanitize_res.items() if info['code']]
-        if invalid_numbers:
-            raise exceptions.UserError(_('Following numbers are not correctly encoded: %s, example : "+32 495 85 85 77, +33 545 55 55 55"', repr(invalid_numbers)))
-        
+        for invalid_number in invalid_numbers:
+            notification_messages.append(_('The following number is not correctly encoded: %s', invalid_number))
```

Comment:
Same remarks for building message at the end, if you update mailing let us update here accordingly.